build/docker/debian.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:1.20-bullseye AS dev
```

Comment:
For consistency, how about we rename Dockerfile to `alpine.Dockerfile` much like this one is called `debian.Dockerfile`? :)